xlators/performance/io-cache/src/page.c

```diff
@@ -803,6 +803,9 @@ ioc_frame_unwind(call_frame_t *frame)
         goto unwind;
     }
 
+    /* gnfs requires op_errno to determine is_eof */
```

Comment:
nit - could be moved to line 799 (and then remove line 802).

Reply:
I'll do whatever you all want. I feel like my contribution was more about the work to find what was breaking. I don't understand the code very well and my C is rusty. So I will take the above nit to be a request. So let me work on that. I'm in better shape now that I can update the PR differently.  Our internal github has a different work flow than this sorry for the rough edges on process!

Reply:
I got backed up with work today; even though ti seems simple I want to re-do my test cases the best I can first. So I will target tomorrow to push something over with the requested change.

Reply:
I apologize for the delay in getting back to you. I repeated the tests with how I interpret the comment above for page.c. My tests (in the notes.txt I attached) pass. 

I will work to get the patch out of the rpm build and into the git tree and update the PR shortly.

Reply:
Please verify I understood what you asked for. I will mark this resolved.